style.css

```diff
@@ -0,0 +1,228 @@
+.mobile-header {
+  display: flex;
+  justify-content: space-between;
+}
+
+#shape {
+  position: relative;
+  width: 74px;
+  height: 74px;
+  left: 28px;
+  top: -9px;
+}
+
+#shape1 {
+  position: absolute;
+  top: 0;
+  left: 0;
+}
+
+#logo {
+  position: absolute;
+  width: 74px;
+  height: 74px;
+  left: 34px;
+  top: 0;
+  background: url(images/Shape.svg);
+  background-blend-mode: normal, overlay, normal;
+}
+
+#mobile-logo {
+  position: absolute;
+  width: 72px;
+  height: 74px;
+  left: 0;
+  top: 0;
+  background: url(images/Shape1.svg);
+  background-blend-mode: normal, overlay, normal;
+}
+
+#mobile-menu {
+  display: flex;
+  height: 78px;
+  align-items: center;
+  width: 20px;
+  background: url(images/ic_menu.png);
+  background-repeat: no-repeat;
+}
+
+.main-header {
+  display: none;
+}
+
+.projects-block h2 {
+  font-size: 4em;
+  font-family: Playfair Display, serif;
+  margin-right: 20px;
+}
+
+#primary-text h1 {
+  font-size: 50px;
+  font-family: Playfair Display, serif;
+  padding: 0 20px;
+  color: #1a2236;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 100%;
+}
+
+.content p {
+  font-family: Poppins, serif;
+  padding: 0 20px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 21px;
+  letter-spacing: -0.01em;
+  margin: 0;
+}
```

Comment:
- [ ] Kindly use the `1A2236` as `colour` for text instead of `black`, because the `black` colour is not consistent with your design and looks bad which results in *a bad user experience* too. Also, I would suggest you use one `p` tag instead of two in order to make your design flawless.

<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-04-29 16-08-16](https://user-images.githubusercontent.com/66476329/235463538-c57e473f-132c-4a23-b899-a605e73ad047.png)

</td>
<td>

![Screenshot from 2023-05-01 19-50-34](https://user-images.githubusercontent.com/66476329/235466196-6a8dbef3-782b-44a4-b0e3-fdb83674f4ce.png)

</td>
</tr>
</table>